### PR TITLE
[SCB-1229] update the version of dependency

### PR DESCRIPTION
**Pull request by heyile**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install -Pit` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

**Comment by liubao68** [comment]
Does license files need upgrated? see https://github.com/apache/servicecomb-java-chassis/pull/1149

**Comment by heyile** [comment]
> upgrated
I see.

**Comment by heyile** [comment]
> 
> 
> Does license files need upgrated? see #1149
I will update license after #1149 .

**Comment by WillemJiang** [comment]
We need to merge this patch and regenerate the license information for it. 
If the third party dependencies are using ASL, it just take few mins to fix the version.
Let's merge this PR first.

File: integration-tests/tracing-tests/pom.xml [hunk 1]
```diff
@@ -66,6 +66,16 @@
     <dependency>
       <groupId>org.apache.servicecomb</groupId>
       <artifactId>tracing-zipkin</artifactId>
+      <exclusions>
```
Comment (wujimin) [inline integration-tests/tracing-tests/pom.xml]: should find the root cause of conflicts

**Comment by WillemJiang** [comment]
If we don't plan to merge it before the java-chassis 1.2.0 release.  I will merge the License fix shortly.

File: integration-tests/tracing-tests/pom.xml (hunk 1 above)
Comment (heyile) [reply integration-tests/tracing-tests/pom.xml]: ok, I see

Comment (heyile) [reply integration-tests/tracing-tests/pom.xml]: **main reason**:
the version 1.7.7 do not have the check of log4j-over-slf4j.jar AND slf4j-log4j12.jar . 
But in the  version 1.2.36 , Log4jLoggerFactory will check it. 

**change**: 
  do not need to exclude all self4j.
```
    <dependency>
      <groupId>org.apache.servicecomb</groupId>
      <artifactId>tracing-zipkin</artifactId>
      <exclusions>
        <exclusion>
          <groupId>org.slf4j</groupId>
          <artifactId>log4j-over-slf4j</artifactId>
        </exclusion>
      </exclusions>
    </dependency>
```

**more deatails**:
https://github.com/qos-ch/slf4j/commit/7a23ca74ae94f87675b25414d6c67a9e9ba9d98f#diff-8c10d2281ed0eaa6d7be5f600d9d673c
https://www.slf4j.org/codes.html#log4jDelegationLoop

File: core/src/main/java/org/apache/servicecomb/core/provider/producer/ProducerProviderManager.java [hunk 2]
```diff
@@ -97,7 +97,12 @@ private void onClose() {
       }
 
       if (Closeable.class.isInstance(operationMeta.getExecutor())) {
-        IOUtils.closeQuietly((Closeable) operationMeta.getExecutor());
+        Closeable executor = (Closeable) operationMeta.getExecutor();
```
Comment (wujimin) [inline core/src/main/java/org/apache/servicecomb/core/provider/producer/ProducerProviderManager.java]: seems they are doing the same thing
why need the change?

File: foundations/foundation-vertx/src/test/java/org/apache/servicecomb/foundation/vertx/http/TestReadStreamPart.java [hunk 3]
```diff
@@ -186,7 +187,7 @@ public void saveToFile() throws InterruptedException, ExecutionException, IOExce
 
     part.saveToFile(file.getAbsolutePath()).get();
 
-    Assert.assertEquals(src, FileUtils.readFileToString(file));
+    Assert.assertEquals(src, FileUtils.readFileToString(file, Charset.defaultCharset()));
```
Comment (wujimin) [inline foundations/foundation-vertx/src/test/java/org/apache/servicecomb/foundation/vertx/http/TestReadStreamPart.java]: replace all defaultCharset to UTF_8?

File: metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/DefaultRegistryInitializer.java [hunk 4]
```diff
@@ -24,14 +24,14 @@
...
 import com.netflix.servo.DefaultMonitorRegistry;
-import com.netflix.spectator.servo.ServoRegistry;
 
 public class DefaultRegistryInitializer implements MetricsInitializer {
   public static final String SERVO_POLLERS = "servo.pollers";
 
   private GlobalRegistry globalRegistry;
 
-  private ServoRegistry registry;
+  @SuppressWarnings("deprecation")
```
Comment (wujimin) [inline metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/DefaultRegistryInitializer.java]: seems that we did not strong depend servo
no need to keep it? change to another one?

File: core/src/main/java/org/apache/servicecomb/core/provider/producer/ProducerProviderManager.java (hunk 2 above)
Comment (heyile) [reply core/src/main/java/org/apache/servicecomb/core/provider/producer/ProducerProviderManager.java]: IOUtils.closeQuietly() is deprecated in commons-io 2.6

File: metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/DefaultRegistryInitializer.java (hunk 4 above)
Comment (heyile) [reply metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/DefaultRegistryInitializer.java]: I have create a new issue about it. https://issues.apache.org/jira/browse/SCB-1245 and now, do not have enough time about it

File: foundations/foundation-vertx/src/test/java/org/apache/servicecomb/foundation/vertx/http/TestReadStreamPart.java (hunk 3 above)
Comment (heyile) [reply foundations/foundation-vertx/src/test/java/org/apache/servicecomb/foundation/vertx/http/TestReadStreamPart.java]: done. replace all defaultCharset with UTF_8

**Pull request merged**